netcore/src/OPADotNet.Automapper/OPADotNet.Automapper.csproj

```
@@ -1,18 +1,18 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
-  </PropertyGroup>
+	<PropertyGroup>
+		<TargetFramework>net6.0</TargetFramework>
```

```suggestion
		<TargetFrameworks>netcoreapp3.1;net6.0</TargetFrameworks>
```

Since we need to be backwards compatible